ospo-mindmap/Content/ospomindmap.md

```diff
@@ -202,11 +202,39 @@ incubation and post launch
   - Legal Network
 - Eclipse Foundation
 - Linux Foundation
-  - FINOS
-  - OpenSSF
-  - OpenChain
-  - SPDX
-  - TODO Group
+  - Open Source Initiatives Adoption, Guidance & Education
```

Comment:
I would not use "Open Source Initiatives" here to not confuse TODO with OSI but something like "Open Source Program Offices".

Reply:
what about "OSPOs and similar open source initiatives"? I want to exclude OSPO since many orgs use different branding (OSO, OS Center) but overall, they act as an OSPO 🤔